packages/components/src/components/hive-footer/index.tsx

```diff
@@ -54,22 +57,11 @@ export const HiveFooter: FC<HiveFooterProps> = ({
             >
               Pricing
             </a>
-            <a
-              className="hive-focus -m-2 rounded p-2 font-medium hover:text-blue-700 hover:underline dark:hover:text-blue-100"
-              href="https://the-guild.dev/contact"
-              onClick={event => {
-                if (window.$crisp) {
-                  window.$crisp.push(['do', 'chat:open']);
-                  event.preventDefault();
-                }
-              }}
-            >
-              Contact Us
-            </a>
+            <ContactUs />
```

Comment:
I feel that the `className` should be here, because it's "footer link style", and the onClick and href are reusable, so maybe something like

```suggestion
            <a
              href="hive-focus -m-2 rounded p-2 font-medium hover:text-blue-700 hover:underline dark:hover:text-blue-100"
              {...CONTACT_LINK}
            />
```

and we could spread the same `CONTACT_LINK` props elsewhere where we have the `if (window.$crisp)` bit.

Reply:
`<a>` contains onClick, passing function from server components (hive-footer is server component) to client components is not possible

Reply:
okay, I looked up all the callsites with `window.$crisp` and refactoring would need something tricky. sorry for bikeshedding, let's ship